modules/images-configuration-insecure.adoc

```diff
@@ -58,7 +58,7 @@ registries are allowed. Either `blockedRegistries` or `allowedRegistries` can be
 <3> Specify registries that should be permitted for image pull and push actions. All other registries are denied. Either `blockedRegistries` or `allowedRegistries` can be set, but not both.
 +
 The Machine Config Operator (MCO) watches the `image.config.openshift.io/cluster`
-for any changes to registries and reboots the nodes when it detects changes.
+for any changes to registries and drains the nodes when it detects changes.
```

Comment:
Perhaps we should specify specifically that:
1. as of 4.7, this change will no longer trigger a reboot (if no other changes were applied at the same time)
2. it will instead drain the nodes, apply the changes and reload crio.service on the host, and then uncordon
3. the `/host/etc/containers/registries.conf` is a bit odd, it should just be `/etc/containers/registries.conf` on the actual hosts. The only time you would see if in `/host...` is if you have a container that mounted the system root to `/host` (e.g. a debug container)